package NYT.NChunkClient.NProto;

import "yt/core/misc/proto/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TBlockId
{
    required NYT.NProto.TGuid chunk_id = 1;
    required int32 block_index = 2;
}

////////////////////////////////////////////////////////////////////////////////